diff -burpBN ../tt.old/src/mapper.c ./src/mapper.c
--- ../tt.old/src/mapper.c	2014-05-11 18:56:42.000000000 -0700
+++ ./src/mapper.c	2014-05-28 23:02:20.225453500 -0700
@@ -108,13 +108,13 @@ DO_COMMAND(do_map)
 			{
 				if (map_table[cnt].check > 0 && ses->map == NULL)
 				{
-					tintin_printf2(ses, "#MAP: This session has no map data. Use #map create or #map read to create one.");
+					tintin_printf(ses, "#MAP: This session has no map data. Use #map create or #map read to create one.");
 
 					return ses;
 				}
 				if (map_table[cnt].check > 1 && ses->map->room_list[ses->map->in_room] == NULL)
 				{
-					tintin_printf2(ses, "#MAP: You are not inside the map. Use #map goto to enter it.");
+					tintin_printf(ses, "#MAP: You are not inside the map. Use #map goto to enter it.");
 
 					return ses;
 				}
@@ -1796,7 +1796,7 @@ DO_MAP(map_undo)
 			tintin_printf2(ses, "#MAP UNDO: Invalid last move.");
 			return;
 		}
-		tintin_printf2(ses, "#MAP UNDO: Moving to room %s.", link->str2);
+		tintin_printf(ses, "#MAP UNDO: Moving to room %s.", link->str2);
 
 		goto_room(ses, atoi(link->str2));
 	}
@@ -2444,6 +2444,11 @@ int follow_map(struct session *ses, char
 		pop_call();
 		return 1;
 	}
+	else
+	{
+		tintin_printf(ses, "#MAP: Unable to move to undefined exit!");
+	}
+
 	pop_call();
 	return 0;
 }
