diff -burpBN ../tt.old/src/mapper.c ./src/mapper.c
--- ../tt.old/src/mapper.c	2013-12-31 22:16:29.000000000 -0800
+++ ./src/mapper.c	2014-05-17 05:33:55.246665100 -0700
@@ -1796,7 +1796,7 @@ DO_MAP(map_undo)
 			tintin_printf2(ses, "#MAP UNDO: Invalid last move.");
 			return;
 		}
-		tintin_printf2(ses, "#MAP UNDO: Moving to room %s.", link->str2);
+		/*tintin_printf2(ses, "#MAP UNDO: Moving to room %s.", link->str2);*/
 
 		goto_room(ses, atoi(link->str2));
 	}
